guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-smart-proxy.adoc

```diff
@@ -45,10 +45,9 @@ You can move the copied file to the applicable path if required.
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {installer-scenario-smartproxy} \
---foreman-proxy-content-parent-fqdn "{foreman-example-com}" \
 --foreman-proxy-register-in-foreman "true" \
 --foreman-proxy-foreman-base-url \ "https://{foreman-example-com}"
```

Comment:
There's also a bug here:
```suggestion
--foreman-proxy-foreman-base-url \ "https://{foreman-example-com}" \
```

Reply:
```suggestion
--foreman-proxy-foreman-base-url "https://{foreman-example-com}" \
```

Like this?

Reply:
Oh yes, I didn't even catch that. Thanks :)